Gruntfile.js

```diff
@@ -253,8 +253,8 @@ module.exports = function(grunt) {
 
       connectionString = 'mongodb://' + authenticationString + config.dbHost + portString + '/' + config.dbName;
     }
-    if(typeof config.authSource === 'string' && config.authSource !== '' ){
-      connectionString += '?authSource=' + config.authSource
+    if(typeof config.dbAuthSource === 'string' && config.dbAuthSource !== '' ){
+      connectionString += '?dbAuthSource=' + config.dbAuthSource
```

Comment:
'?dbAuthSource=' should remain as '?authSource='. [ref](https://docs.mongodb.com/manual/reference/connection-string/#urioption.authSource)